inlang/source-code/sdk/src/v2/types.ts

```diff
@@ -0,0 +1,122 @@
+import { LanguageTag as Locale } from "@inlang/language-tag"
```

Comment:
You can pull the Locale type into the types here. 

- language tag has been it's own package to allow the community to use it 
-- this is not happening
- eases getting rid of versioned-interfaces